gammapy/modeling/models/spectral.py

```diff
@@ -1736,18 +1739,21 @@ def write(self, overwrite=False):
 
     @classmethod
     def from_dict(cls, data):
+        data = data["spectral"]
         filename = data["filename"]
         m = RegionNDMap.read(filename)
         model = cls(m, filename=filename)
         for idx, p in enumerate(model.parameters):
-            p.value = data["spectral"]["parameters"][idx]["value"]
+            par = p.to_dict()
```

Comment:
Same comment here modifying the constructor to properly take into account the norm might prove simpler.